src/api/protectedApiClient.ts

```diff
@@ -74,9 +83,9 @@ const registerTickets = (request: RegisterTicketsRequest) => {
 };
 
 const getMyEvents = (): Promise<EventInformation[]> => {
-  return AppAxiosInstance.get(ProtectedApiClientRoutes.MY_EVENTS).then(
-    (res) => res.data.events,
-  );
+  return AppAxiosInstance.get(
+    `${ProtectedApiClientRoutes.EVENTS}/signed_up`,
+  ).then((res) => res.data.events);
```

Comment:
This had nothing to do w this issue, just was bothering me that it was inconsistent